torch/distributed/_tensor/device_mesh.py

```diff
@@ -1,4 +1,4 @@
-from torch.distributed._device_mesh import (  # noqa: F401
+from torch.distributed.device_mesh import (  # noqa: F401
```

Comment:
We should probably return early if `torch.distribtued.is_avaialble()` is False, to satisfy public_api test